modules/lvgl/lvgl_display.c

```diff
@@ -85,9 +85,15 @@ int set_lvgl_rendering_cb(lv_disp_drv_t *disp_drv)
 #endif
 		break;
 	case PIXEL_FORMAT_RGB_888:
+#if defined(CONFIG_RENESAS_RA_DISPLAY)
```

Comment:
Same story here. It seems like the display is using ARGB8888?

Reply:
This change has been reverted